packages/ui-components/src/components/switch-button/test/switch-button.e2e.ts

```diff
@@ -26,6 +26,8 @@ describe('Switch Button (end-to-end)', () => {
 
 				const buttonElement = await page.find('kv-switch-button >>> .switch-button');
 				await buttonElement.click()
+
+				await page.waitForTimeout(300)
```

Comment:
The linter don't catch the ; at the end?

Reply:
@miguelpinto98 I recreate this system in this branch cause I was deleted this, so, the content is exact the same of dev branch and I think that I can't fix anything here.

Reply:
you can add the trailing comma that are missing in the file... In other hand, the `waitForTimeout` is added in this change.